activiti-spring-boot-starter/src/test/java/org/activiti/spring/boot/tasks/TaskRuntimeAppVersionTest.java

```diff
@@ -0,0 +1,54 @@
+package org.activiti.spring.boot.tasks;
+
+import org.activiti.api.runtime.shared.query.Page;
+import org.activiti.api.runtime.shared.query.Pageable;
+import org.activiti.api.task.model.Task;
+import org.activiti.api.task.model.builders.TaskPayloadBuilder;
+import org.activiti.api.task.runtime.TaskRuntime;
+import org.activiti.spring.boot.security.util.SecurityUtil;
+import org.activiti.spring.boot.test.util.TaskCleanUpUtil;
+import org.junit.After;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.NONE)
+public class TaskRuntimeAppVersionTest {
```

Comment:
I know that most of the classes under this package are not following the convention, but it's better to use `IT` suffix instead of `Test` as it's an integration test.

Reply:
Changed it.